### ISSUE TITLE
MacOS 0.70.122 no longer blocks any ads. new release needed.

### ISSUE THREAD

**Issue by CommieChan** [gab-ai-inc/defiant-browser#41]
MacOS version is obsolete, version 0.70.122 no longer effectively block any ads.  A new version is desperately needed on the MacOS platform. 

Go on youtube, splash pages full of ads, and videos will show ads. 
This is not an issue for 1.5.114 under windows.

**Comment by jase52476** [comment]
Yes, I am also having this issue with YouTube specifically.  I noticed that now, ads play on monetized channels, and there's no skip button.  So some ads take over 2 minutes, and it's impossible to get past them.  

Also, I switched over to Brave, updated it (Version 1.10.97 Chromium: 83.0.4103.116 (Official Build) (64-bit)), and it's also having ads but at least they can be skipped now after the button appears.
